app/ConfigValidation/ValidateConfigApplication.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\App\ConfigValidation;
+
+use Symfony\Component\Console\Application;
```

Comment:
The new folder on the top level (ie cli/) makes more sense to me in case we see this as a general entry point to the application (ie a counterpart to the web one in app/) as opposed to a script just part of the build/CI.